l2-contracts/contracts/bridge/L2SharedBridge.sol

```diff
@@ -45,7 +50,7 @@ contract L2SharedBridge is IL2SharedBridge, IL2SharedBridgeLegacy, Initializable
     IL2StandardDeployer public standardDeployer;
 
     /// @dev A mapping l2 token address => l1 token address
-    mapping(bytes32 assetInfo => address assetAddress) public override assetAddress;
+    mapping(bytes32 assetInfo => address assetAddress) public override assetAddress; // ToDo: Why do we call asset if it's token?? and it's not set
```

Comment:
It might be a general asset, not just a token. It should be set somewhere, I did not get around to that